components/infobox/wikis/ageofempires/infobox_person_player_custom.lua

```diff
@@ -99,8 +99,8 @@ function CustomPlayer.run(frame)
 	if Logic.readBool((_args.autoPI or ''):lower()) then
 		local _, roleType = CustomPlayer._getRoleType(_args.roleList)
 
-		autoPlayerIntro = PlayerIntroduction._main{
-			playerInfo = 'direct',
+		autoPlayerIntro = PlayerIntroduction{
```

Comment:
my intended entry point was `.run` (would not need the :create()) but this is fine for me too

Reply:
Already changed before your comment came through :D